cmd/root.go

```diff
@@ -33,6 +37,76 @@ func toRunE(main commandMain) runE {
 	}
 }
 
+// GA => Google Analytics
+func registerInvocationToGA(cmd *cobra.Command, socketPath string) {
+	go func() {
+		// Errors are reported in the server logs, so no need to expose them
+		// to the user
+		name := cmd.Name()
+		if name == "server" {
+			name = "wash"
+		}
+		// We use client.ForUNIXSocket instead of cmdutil.NewClient because the socketPath
+		// could change. The latter's possible if cmd.Name() == "wash", i.e. the root
+		// command.
+		_ = client.ForUNIXSocket(socketPath).Screenview(name, analytics.Params{})
+	}()
+}
+
+// GA => Google Analytics
+func waitForGARegistration() {
```

Comment:
This seems like it should wait for the GA flush to complete, or FlushDuration, whichever comes first. Not the full FlushDuration.

Reply:
Yeah, makes sense. I'll add that.